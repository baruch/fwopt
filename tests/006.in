-A INPUT -p 47 -j DROP
-A INPUT -p udp -j ACCEPT
-A INPUT -p tcp -j ACCEPT
