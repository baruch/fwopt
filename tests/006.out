iptables -A INPUT -p 47 -j DROP
iptables -A INPUT -p udp -j ACCEPT
iptables -A INPUT -p tcp -j ACCEPT
